### ISSUE TITLE
Update CIDRs

### ISSUE THREAD

**Issue by carlbennett** [carlbennett/bnetdocs-web#86]
The file below needs the new load balancers added for Classic Battle.net.

- connect.classic.blizzard.com
- connect-bot.classic.blizzard.com
- connect-bot-na.classic.blizzard.com
- connect-use.classic.blizzard.com
- connect-usw.classic.blizzard.com
- connect-kor.classic.blizzard.com
- connect-eur.classic.blizzard.com
- connect-forever.classic.blizzard.com

https://github.com/BNETDocs/bnetdocs-web/blob/0a93306134911a5eeac2d53bc31f07bb5a0e75fc/src/static/a/Blizzard-CIDRs.txt

**Comment by carlbennett** [comment]
```
connect.classic.blizzard.com is an alias for connect-na.classic.blizzard.com.
connect-na.classic.blizzard.com has address 37.244.27.170
connect-bot.classic.blizzard.com is an alias for connect-bot-na.classic.blizzard.com.
connect-bot-na.classic.blizzard.com has address 37.244.27.110
connect-use.classic.blizzard.com has address 37.244.27.100
connect-usw.classic.blizzard.com has address 37.244.26.100
connect-kor.classic.blizzard.com has address 117.52.35.100
connect-eur.classic.blizzard.com has address 37.244.29.100
```

**Comment by carlbennett** [comment]
```
connect-forever.classic.blizzard.com has address 37.244.23.238
```

**Comment by carlbennett** [comment]
`connect-live-emea-b.classic.blizzard.com has address 37.244.28.169`

**Comment by carlbennett** [comment]
`connect-emea.classic.blizzard.com has address 37.244.28.170`

Depending on geographic location:
`connect.classic.blizzard.com is an alias for connect-na.classic.blizzard.com.`
`connect-na.classic.blizzard.com has address 37.244.27.70`

BogdanW3 reports `connect.classic.blizzard.com` is an alias of `connect-emea.classic.blizzard.com`

**Comment by carlbennett** [comment]
This issue has been resolved as of 0169940466077cd212c38bcc022f148f813d1282.

**Comment by carlbennett** [comment]
This issue has been resolved as of e710ec680eae4962b8ae539b2a91f8661f5156a9...0169940466077cd212c38bcc022f148f813d1282.
